### ISSUE TITLE
Disable the header modification

### ISSUE THREAD

**Issue by jsmad** [schweikert/postgrey#31]
Is it possible to disable the modification of the email headers?

I'd like the email to be left intact, without adding the X-Greylist line.

Thank you.

**Comment by budwig** [comment]
Yes it's possible. Add this line to /etc/postfix/header_checks file:
/^X-Greylist::/   IGNORE
And in your main.cf add this: 
header_checks = regexp:/etc/postfix/header_checks

Restart postfix.

**Comment by jsmad** [comment]
Nice and simple, thank you!

Although postgrey should have that as optional, in my humble opinion. As an alternative, there is the new postscreen feature of postfix.

**Comment by budwig** [comment]
Glad it helped. And yes I agree that it should be an option to disable adding that header. Nevertheless, using header_checks grants more flexibility, as you can customize it for other headers filtering.

About postscreen, it is a completely different concept from greylisting (postgrey). Personnaly, I use both on my server; this combination virtually eliminated 99% of the spam!
